apps/complaints/acceptance/pages/confirmation.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = {
-  url: 'confirmation'
+  url: 'confirmation',
+  content: {
+    sent: 'The complaint has been sent.',
```

Comment:
Normally I would not check for content of a page but this is a exception because I will be checking the Banner content to make sure it is there. I will refactor using the translation for this.